app/src/main/java/com/javahelp/frontend/MyViewModel.java

```diff
@@ -0,0 +1,48 @@
+package com.javahelp.frontend;
+
+import android.app.Application;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class MyViewModel extends AndroidViewModel {
+    private MutableLiveData<String> userName;
+    private MutableLiveData<String> passWord;
+
+    public MyViewModel(@NonNull Application application) {
+        super(application);
+    }
+
+
+    public void setUserName(String username) {
+        userName.setValue(username);
+
+    }
+
+    public MutableLiveData<String> getUserName() {
+        if (userName == null) {
```

Comment:
In my `[Backend] Register User Use Case` will handle the null case for username to make sure that anyone has a valid name, and not just null. Is this or that a better approach?